README.md

```diff
@@ -65,7 +65,7 @@ It should print a following string `"{name} is creating an API..."`
 where `name` is the `name` of engineer and return address of the API: `"http://127.0.0.1:8000"`.
 
 ```python
-address = backend_dev.create_powerful_api()  # "Bob is creating an API..."
```

Comment:
Remove your changes from the `README.md` file.

Reply:
Changes were not removed from the `README.md` file. Make sure that this file together with `__init__.py` not included in the PR.